projects/demo/src/modules/charts/bar-chart/bar-chart.template.html

```diff
@@ -31,6 +31,16 @@
             heading="Same values with collapsed mode"
             [content]="example2"
         >
+            <tui-notification class="tui-space_bottom-6">
+                Use
+                <a
+                    routerLink="/directives/hint-controller"
+                    tuiLink
+                >
+                    <code>HintController</code>
```

Comment:
Let's add this controller to this component ?

https://github.com/Tinkoff/taiga-ui/blob/main/projects/demo/src/modules/app/autogenerated-code-hints/autogenerated-code-hints.component.ts